apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -465,6 +474,25 @@ const aichatSlice = createSlice({
       state.fieldVisibilities =
         levelAichatSettings?.visibilities || DEFAULT_VISIBILITIES;
     },
+    resetToDefaultAiCustomizations: (
+      state,
+      action: PayloadAction<{
+        levelAichatSettings?: LevelAichatSettings;
```

Comment:
Oh, and does this need to be inside an object? Or just pass the levelAichatSettings directly?